action/well_known.go

```diff
@@ -0,0 +1,60 @@
+package action
```

Comment:
I think we should avoid underscore in file names and follow the same pattern as packages (see https://stackoverflow.com/a/39625565 )